crhchttp/http_cloud_dot_redhat.go

```diff
@@ -129,15 +130,10 @@ func GetClient(authConfig *AuthConfig) HTTPClient {
 		caCertPool := x509.NewCertPool()
 		caCertPool.AppendCertsFromPEM(caCert)
 
-		transport := &http.Transport{TLSClientConfig: &tls.Config{RootCAs: caCertPool}}
-		return &http.Client{
-			Timeout:   30 * time.Second,
-			Transport: transport,
-		}
+		transport.TLSClientConfig = &tls.Config{RootCAs: caCertPool}
 	}
-	log.Info("configured to not using the certificate for this request")
```

Comment:
why remove this log?

Reply:
1. this function now only has 1 return statement. If we have `validate_cert: true`, we would still log this message.
2. not sure how useful this message is anyway. We can see in the CR if `validate_cert: false`, so we are not really losing anything here.